scripts/openmc-plot-mesh-tally

```diff
@@ -0,0 +1,325 @@
+import tkinter.messagebox as messagebox
+import tkinter.ttk as ttk
+
+import matplotlib
+matplotlib.use("TkAgg")
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg
+from matplotlib.backends.backend_tkagg import NavigationToolbar2Tk
+from matplotlib.figure import Figure
+import matplotlib.pyplot as plt
+import numpy as np
+
+from openmc import StatePoint, MeshFilter
+
+
+class MeshPlotter(tk.Frame):
+    def __init__(self, parent, filename):
+        tk.Frame.__init__(self, parent)
+
+        self.labels = {'Cell': 'Cell:', 'Cellborn': 'Cell born:',
+                       'Surface': 'Surface:', 'Material': 'Material:',
+                       'Universe': 'Universe:', 'Energy': 'Energy in:',
+                       'Energyout': 'Energy out:'}
+
+        self.filterBoxes = {}
+
+        # Read data from source or leakage fraction file
+        self.get_file_data(filename)
+
+        # Set up top-level window
+        top = self.winfo_toplevel()
+        top.title('Mesh Tally Plotter: ' + filename)
+        top.rowconfigure(0, weight=1)
+        top.columnconfigure(0, weight=1)
+        self.grid(sticky=tk.W+tk.N)
+
+        # Create widgets and draw to screen
+        self.create_widgets()
+        self.update()
+
+    def create_widgets(self):
+        figureFrame = tk.Frame(self)
+        figureFrame.grid(row=0, column=0)
+
+        # Create the Figure and Canvas
+        self.dpi = 100
+        self.fig = Figure((5.0, 5.0), dpi=self.dpi)
+        self.canvas = FigureCanvasTkAgg(self.fig, master=figureFrame)
+        self.canvas.get_tk_widget().pack(side=tk.TOP, fill=tk.BOTH, expand=1)
+
+        # Create the navigation toolbar, tied to the canvas
+        self.mpl_toolbar = NavigationToolbar2Tk(self.canvas, figureFrame)
+        self.mpl_toolbar.update()
+        self.canvas._tkcanvas.pack(side=tk.TOP, fill=tk.BOTH, expand=1)
+
+        # Create frame for comboboxes
+        self.selectFrame = tk.Frame(self)
+        self.selectFrame.grid(row=1, column=0, sticky=tk.W+tk.E)
+
+        # Tally selection
+        labelTally = tk.Label(self.selectFrame, text='Tally:')
+        labelTally.grid(row=0, column=0, sticky=tk.W)
+        self.tallyBox = ttk.Combobox(self.selectFrame, state='readonly')
+        self.tallyBox['values'] = [self.datafile.tallies[i].id
+                                   for i in self.meshTallies]
+        self.tallyBox.current(0)
+        self.tallyBox.grid(row=0, column=1, sticky=tk.W+tk.E)
+        self.tallyBox.bind('<<ComboboxSelected>>', self.update)
+
+        # Planar basis selection
+        labelBasis = tk.Label(self.selectFrame, text='Basis:')
+        labelBasis.grid(row=1, column=0, sticky=tk.W)
+        self.basisBox = ttk.Combobox(self.selectFrame, state='readonly')
+        self.basisBox['values'] = ('xy', 'yz', 'xz')
+        self.basisBox.current(0)
+        self.basisBox.grid(row=1, column=1, sticky=tk.W+tk.E)
+        self.basisBox.bind('<<ComboboxSelected>>', self.update)
```

Comment:
`<<ComboboxSelected>>` should be defined as constant, it's used in several places. Avoids typos.